config/orm.js

```diff
@@ -0,0 +1,75 @@
+const db = require('./connection.js');
+const userData = mongoose.model("userData", userSchema);
+
+module.exports = {
+
+    createUser: function (req, res) {
+        db.userData
+        .create({username: x, password: y, email: z })
+        .then(userData => res.send(userData))
+        .catch(err => console.log(err));
+    },
+    loginUser: function (req, res) {
+        db.userData
+        .findOne ({username: x}, {password: req.body.data})
```

Comment:
since we are looking for a record in the database that matches the data passed in,
```
.findOne({username: req.body.data.username}) 
```